langchain/embeddings/openai.py

```diff
@@ -461,7 +463,7 @@ async def _aembedding_func(self, text: str, *, engine: str) -> List[float]:
             )["data"][0]["embedding"]
 
     def embed_documents(
-        self, texts: List[str], chunk_size: Optional[int] = 0
+        self, texts: List[str], chunk_size: Optional[int] = 0, **kwargs
```

Comment:
this signature is defined on base embeddings class, probably don't want to break the interface just here and im not sure we should add it to the base class.

what if added something like this instead? https://github.com/hwchase17/langchain/blob/c7b687e944883df972cabdf00064112587306daf/langchain/llms/openai.py#L137

Reply:
for embeddings you can already pass model in directly, no? 
```python
OpenAIEmbeddings(model=...)
```

Reply:
yes but I need to place it also on the request body because that is parsed by our custom proxy 😢